qemu/tests/balloon_check.py

```diff
@@ -163,10 +183,11 @@ def get_memory_boundary(self, balloon_type=''):
         if self.params.get('os_type') == 'windows':
             logging.info("Get windows miminum balloon value:")
             self.vm.balloon(1)
-            time.sleep(90)
+            balloon_timeout = self.params.get("balloon_timeout", 9600)
```

Comment:
I set it to 9600s because I met it when sometimes it took more than 2000s to complete the balloon. But I retest it with latest driver, seems 900s is long enough now.  I've update the timeout, please help review again, thanks!